compiler/testData/codegen/bytecodeText/forLoop/primitiveProgression.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: JVM_IR
```

Comment:
Could you add comments to the tests that now fail on JVM_IR, so it is easy for us to see why these fail and if that is by design?